src/services/postService.js

```diff
@@ -104,6 +108,41 @@ module.exports = {
     return client.sendAsync(GET_BLOG_ENTRIES, params, getHundredxPosts);
   },
 
+  deletePost: function({ permLink }) {
+    return PostModel.findOne({
+      where: { permLink },
+    }).then(postInOurDb => {
+      const keyVal = {};
+      keyVal[DELETED] = true;
+      return postInOurDb.update(keyVal);
+    }).catch(err => {
+      throw new Error(err.error_description);
```

Comment:
Could we pass the entire error object in rather than just the `error_description`? There may be other useful data for debugging

Reply:
done